src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not center text in a button, even if the button is wide.">
+
+![Do not center text in a button, even if the button is wide.](images/button_usage_9_test.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Alignment
+
+Alignment refers to whether the buttons are aligned to the right or the left of a window, container, or layout. Buttons are unique, more so than any other component, in that their alignment depends on where they appear and whether or not they’re contained within another component.
+
+As a general rule, on full-page designs, the primary button is on the left side of the page. When the browser window is large and the user is scrolling to read, it’s best to have the primary button where the user’s attention has been focused all along. Whereas in wizards, where a user is progressing through a series of steps or dialog windows, the primary action traditionally sits at the bottom right. Buttons within components such as notifications, search fields, and data tables are also right-aligned.
 
-<Caption>Secondary / Primary button pair in a modal</Caption>
+In some cases a button group—or even a single button in the case of a side panel or small tile—may span the entire width of a window or container. Typical nested button locations include:
+
+- Banner calls to action
+- Dialog windows
+- Wizards
+- Forms
+- Cards
+- Toolbars
+
+<Row>
+<Column colLg={8}>
+
+![Primary button context](images/button_usage_10.png)
 
 </Column>
 </Row>
 
-If there is a third action, place this within the modal itself as a link, rather than alongside the Secondary / Primary buttons.
+| Alignment         | Use case                                                                                                           |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------ |
+| _Left-justified_  | Banner call to actions, in-page forms, and nested buttons in components like tiles                                  |
+| _Right-justified_ | Inline notifications, inline field buttons and  data tables, progressive forms, wizards, and single-button dialogs |
+| _Full span_       | Dialogs, side panels, and small tiles (optional treatment)                                                          |
+
+#### Fluid versus fixed buttons
+
+Button alignment is also closely related to whether the button is treated as a fixed or a fluid element within a layout. When we say “fluid”, we mean that the button becomes a part of a larger, compound component by bleeding to two or more edges of its container. Rather than defining the a fluid button’s width in columns or mini units, its width is defined as a percentage (often 50%) of the container’s width. Also, as a general rule, fluid buttons are never left-aligned in a layout or a container — they’re always either right-aligned, or span the full width of the container.
```

Comment:
Talking about a fluid button here can be confusing as we don't have a way to refer to this in code. Similar to the full bleed stuff, might make sense to try and ship this functionality together otherwise we end in a situation where design and dev are mismatched when building.

Reply:
Again, we need language to talk about this, because it exists in product and in a component that we ship and I consider it a foundational visual characteristic of the system.